libtcmu_priv.h

```diff
@@ -20,11 +20,7 @@
 #include <gio/gio.h>
 #include <pthread.h>
 
-#include "scsi_defs.h"
 #include "darray.h"
-#include "ccan/list/list.h"
-#include "tcmur_aio.h"
-#include "tcmu-runner.h"
 
```

Comment:
This breaks the build. Doing a make results in this error:

> /root/targetcli/devel/tcmu-runner/api.c: In function ‘tcmu_emulate_evpd_inquiry’:
> /root/targetcli/devel/tcmu-runner/api.c:627:15: error: dereferencing pointer to incomplete type
>    if (rhandler->unmap) {
>                ^
> /root/targetcli/devel/tcmu-runner/api.c:724:15: error: dereferencing pointer to incomplete type
>    if (rhandler->unmap)
> 

I think you need to move some more stuff around like add a include tcmu-runner.h somewhere. Or maybe drop this patch until Xiubo has done his surgery on the code to separate/move the lib stuff and tcmu-runner stuff.

Reply:
OK, Xiubo will finish his work very soon so drop this patch and wait for his PR.

Reply:
Oh wait. I think there was a small misunderstanding. I was just asking you to drop that one patch.

I think the other patches are ok.

Reply:
Yeh I misunderstood it but that's ok, these patches have been merged together with Xiubo's change and he opens a new PR. :-)